docs/guide/custom-functions.md

```diff
@@ -306,6 +319,10 @@ export const MyFunctionNameTranslations = {
 HyperFormula.registerFunctionPlugin(MyFunctionPlugin, MyFunctionNameTranslations);
 ```
 
+::: tip
+Before using a translated function name, remember to [register and set the language](localizing-functions.md).
+:::
+
```

Comment:
During testing I decided that we should mention that.

Reply:
ok